text/0000-new-buildpack-toml-keys.md

```diff
@@ -0,0 +1,88 @@
+# Meta
+[meta]: #meta
+- Name: New Buildpack Descriptor Keys
+- Start Date: 2020-12-12
+- Author(s): [@jkutner](@jkutner)
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+This is a proposal for new keys in the `buildpack.toml` file.
+
+# Definitions
+[definitions]: #definitions
+
+* _buildpack descriptor_ - The `buildpack.toml` file that describes a buildpack.
+
+# Motivation
+[motivation]: #motivation
+
+New optional fields in the `buildpack.toml` would allow us to introduce more information into the Buildpack Registry search API and the web interface.
+
```

Comment:
`description` and licenses would also be valuable output from `pack inspect-buildpack`.

Reply:
Standardization is a big pro for me.